### ISSUE TITLE
Cannot compile for i386 under debian 64bit

### ISSUE THREAD

**Issue by Darker** [log4cplus/log4cplus#109]
I have the nasty issue that I need to build both x64 and x32 versions of the latest release I downloaded from SourceForge. 64bit version works perfectly, but the 32bit build just isn't 32bit. What I did to compile was:

    export CXXFLAGS="-std=c++0x -m32"

Although size of the output files was slightly different, they were still 64bit. Is it possible to achieve my goal?

**Comment by wilx** [comment]
Well, if `-m32` does not work then I would suggest true cross compiler. Also, `-m32` might be failing because you are missing some some development related packages. If everything fails, install 32bit OS into a VM and compile there. :)

**Comment by Darker** [comment]
Actually I spent this whole week learning how to compile other project in multi-arch mode so I don't want to give up now. With the other project I have suceeded and if I fail with log4cplus I will fall back to already existing 32bit binaries that are used on 32bit release server. But I was hoping to make complete install package x64/x32 with the latest log4cplus.

Is there such thing as cross compiler on linux?
